blazor/heatmap-chart/axis.md

```diff
@@ -359,4 +359,119 @@ Axis label increment in the heat map is used to display the axis labels with reg
 
 ```
 
-![Incrementing Axis Label in Blazor HeatMap Chart](images/axis/blazor-heatmap-chart-increment-axis-label.png)
\ No newline at end of file
+![Incrementing Axis Label in Blazor HeatMap Chart](images/axis/blazor-heatmap-chart-increment-axis-label.png)
+
+## Multilevel labels
+
+Multilevel labels are used to categorize a group of axis labels as a specific category, displayed using a label. You can add many levels of such labels by adding multiple `HeatMapMultiLevelLabel` tags.
```

Comment:
The APIs in this content already have API links. But they are not linked here. Remove the code style and add the links.